lib/object.c

```diff
@@ -30,19 +30,23 @@ static tool_rc tpm2_util_object_load2(
             const char *objectstr,
             const char *auth,
             bool do_auth,
-            tpm2_loaded_object *outobject) {
+            tpm2_loaded_object *outobject,
+            bool is_restricted_pswd_session) {
```

Comment:
isn't this TRUE if esys context is NULL? So can't we just determine based on ESYS context?

Reply:
Yes and no. Because esys context is never passed as NULL as it is required in do_ctx_file  OR tpm2_util_sys_handle_to_esys_handle  routines, which ever path it takes.